src/tests/js/test-router-reqview.js

```diff
@@ -0,0 +1,35 @@
+// Test router module to expose the "req" object so that we can confirm that our middleware is installed correctly.
+"use strict";
+var namespace = "gpii.express.tests.router.reqview";
+var fluid     = fluid || require('infusion');
+var gpii      = fluid.registerNamespace("gpii");
+var reqView   = fluid.registerNamespace(namespace);
+
+reqView.addRoutesPrivate = function(that) {
```

Comment:
As before - name this descriptively. We no longer believe in the concept of "private" implementation, just implementation for a particular nameable purpose.